include/eve/module/core/regular/impl/try_each_group_position.hpp

```diff
@@ -0,0 +1,48 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/module/core/regular/slide_right.hpp>
+
+namespace eve::detail
+{
+
+template<simd_value T, std::ptrdiff_t G, std::size_t... pos>
+EVE_FORCEINLINE auto
+try_each_group_position_default(T x, eve::fixed<G>, std::index_sequence<pos...>) noexcept
+{
+  return kumi::tuple {eve::slide_right(x, x, eve::index<pos * G>)...};
```

Comment:
can't pos * G overflow the size of x ? Or is it what you try to achieve here ?

Reply:
That was an old version - sorry - some rebasing got in the way.